config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java

```diff
@@ -178,28 +174,27 @@ public String doGetConfig(HttpServletRequest request, HttpServletResponse respon
                         && cacheItem.getIps4Beta().contains(clientIp)) {
                     isBeta = true;
                 }
-                
```

Comment:
please don't change the indent

Reply:
Sorry, I didn't pay attention to the indentation issue. I have adjusted it now.